### PR TITLE
Postgres docker

### PR DESCRIPTION
**Pull request by JC1738**
This is a way to run postgres and liquibase without interfering with Make file or others.  People can specialize there taskfile.yml:  https://github.com/go-task/task

### CODE REVIEW

**Comment by sous-machine-account** [comment]
LGTM

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: Taskfile.yml [hunk 1]
```diff
@@ -0,0 +1,117 @@
+version: '2'
```
Comment (nyarly) [inline Taskfile.yml]: I'm not clear on what advantages we get out of adding a different build tool to the project.

File: Makefile [hunk 2]
```diff
@@ -73,9 +73,12 @@ COVER_DIR := /tmp/sous-cover
 TEST_VERBOSE := $(if $(VERBOSE),-v,)
 SOUS_PACKAGES:= $(shell go list -f '{{.ImportPath}}' ./... | grep -v 'vendor')
 SOUS_PACKAGES_WITH_TESTS:= $(shell go list -f '{{if len .TestGoFiles}}{{.ImportPath}}{{end}}' ./...)
+SOUS_TC_PACKAGES=$(shell docker run --rm -v $(PWD):/go/src/github.com/opentable/sous -w /go/src/github.com/opentable/sous golang:1.10 go list -f '{{if len .TestGoFiles}}{{.ImportPath}}{{end}}' ./... | sed 's/_\/app/github.com\/opentable\/sous/')
```
Comment (nyarly) [inline Makefile]: It seems like this is adding a Go docker container to run commands in. Is that right? What advantage does that give us over running Go natively?

File: Taskfile.yml (hunk 1 above)
Comment (nyarly) [reply Taskfile.yml]: It seems like there's a `task` binary added to the repo. Is that built from this?

File: Makefile (hunk 2 above)
Comment (JC1738) [reply Makefile]: Teamcity agents don't have GO natively.  So with this, don't have to install anything on the agent.  Second advantage though not as much is easier to rev versions.  @nyarly

File: Taskfile.yml (hunk 1 above)
Comment (JC1738) [reply Taskfile.yml]: Task binary is just for the teamcity agents.  It's easy to install task locally, but the TC agents are super clean, so no go, no easy way to get it.  It's a linux bin.

**Pull request merged**